src/Orchestration/Adapter.php

```diff
@@ -67,10 +113,11 @@ abstract public function list(array $filters = []): array;
      * @param string[] $volumes
      * @param array<string, string> $vars
      * @param string $mountFolder
+     * @param string $hostname
      * 
      * @return string
      */
-    abstract public function run(string $image, string $name, array $command, string $entrypoint = '', string $workdir = '/', array $volumes = [], array $vars = [], string $mountFolder = '', array $labels = []): string;
+    abstract public function run(string $image, string $name, array $command = [], string $entrypoint = '', string $workdir = '', array $volumes = [], array $vars = [], string $mountFolder = '', array $labels = [], string $hostname = ''): string;
```

Comment:
You can split the argument list into multiple lines as we discussed in the other sections 🙂